test/handlers/ExtensibleFallbackHandler.spec.ts

```diff
@@ -628,6 +647,16 @@ describe("ExtensibleFallbackHandler", () => {
                 const { validator } = await setupTests();
                 expect(await validator.supportsInterface.staticCall("0x01ffc9a7")).to.be.true;
             });
+
+            it("should use less than 30.000 gas", async () => {
+                const { validator, erc165Bencher } = await setupTests();
+
+                for (const interfaceId of ["0x01ffc9a7", "0xdeadbeef"]) {
+                    const [cold, warm] = await erc165Bencher.supportsInterfaceGas(await validator.getAddress(), interfaceId);
+                    expect(cold).to.be.lessThan(30000);
+                    expect(warm).to.be.lessThan(30000);
+                }
+            });
```

Comment:
Will this pass on Zksync? I know it is failing right now, but it would be good to check as they might use different gas metering

Reply:
I'm running the tests now, will let you know.

Reply:
zkSync tests are borked ATM, so no way to tell :shrug:

Reply:
I think because zkSync has a different gas schedule, it makes sense to disable this test for now. Thanks for the suggestion.